packages/connect/src/api/firmware/getBinaryForFirmwareUpgrade.ts

```diff
@@ -66,3 +66,13 @@ export const getBinaryForFirmwareUpgrade = ({
 
     return getBinary({ release: releaseByFirmware, baseUrl, btcOnly });
 };
+
+const ALL_SLASHES_AT_THE_END_REGEX = /\/+$/;
```

Comment:
https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/api/firmware/getBinary.ts#L4

Reply:
Yes, that is just copy paste for development. I would get only one of those at the end.